crypto/BRCryptoWalletManager.c

```diff
@@ -138,6 +138,8 @@ cryptoWalletManagerCreate (BRCryptoCWMListener listener,
                                              cwmPath,
                                              cryptoNetworkGetHeight(network));
 
+            BRWalletManagerStart (cwm->u.btc);
+
```

Comment:
No, because we call back with the wallet inline in `BRWalletManagerNew`. When that changes, we will need to behave like ETH. See the comment in `BRWalletManagerNew` for details.

Reply:
Okay, I thought this change would include that change - but didn't confirm in the code.